framework/doc/content/source/kernels/ConservativeAdvection.md

```diff
@@ -22,27 +22,97 @@ element solution of the weak formulation. The first term is the volumetric term
 is a surface term describing the advective flux out of the
 volume. `ConservativeAdvection` corresponds to the former volumetric term.
 
-The corresponding Jacobian is given by
+Without numerical stabilization the corresponding Jacobian is given by
 
 \begin{equation}
 \frac{\partial R_i(u_h)}{\partial u_j} = -(\nabla \psi_i, \vec{v} \phi_j).
 \end{equation}
 
+## Full upwinding
+
+Advective flow is notoriously prone to physically-incorrect overshoots
```

Comment:
It appears to me that this scheme works only with continuous FEM, so maybe somewhere adding this restriction will be good.

Reply:
True, thanks.  Done